{"name": "test_login_error_format[i0]", "status": "passed", "attachments": [{"name": "log", "source": "62228af5-4bba-4949-9d36-b907a2854d9e-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "i", "value": "{'user': '1382516192', 'pwd': 'lemon123', 'except': '请输入正确的手机号'}"}], "start": 1585396282418, "stop": 1585396306455, "uuid": "8f436dba-a5f7-4899-a7aa-6974d9f1fb52", "historyId": "44ddca85ea63f38ea6a87a9be719dc00", "testCaseId": "e5c5786b48565192a448e6afdd06dcd9", "fullName": "case_test.test_login.TestLogin#test_login_error_format", "labels": [{"name": "parentSuite", "value": "case_test"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-6P2SG9M"}, {"name": "thread", "value": "20804-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "case_test.test_login"}]}